sql/driver/wire.proto

```diff
@@ -47,10 +47,9 @@ message Datum {
     double float_val = 3;
     bytes bytes_val = 4;
     string string_val = 5;
-    Timestamp time_val = 6;
-    // TODO(vivek): Add additional types like Date and Interval that are
-    // supported by Cockroach but not supported by the Go sql driver. These
-    // will be useful for drivers in other languages.
+    Timestamp date_val = 6;
```

Comment:
We could also choose to use a int64 here since a date doesn't need nanoseconds. But I'm also happy leaving it in as a timestamp type to make the code more readable.